homeassistant/components/konnected/strings.json

```diff
@@ -102,7 +102,8 @@
       "bad_host": "Invalid Override API host URL"
     },
     "abort": {
-      "not_konn_panel": "Not a recognized Konnected.io device"
+      "not_konn_panel": "Not a recognized Konnected.io device",
+      "connection_error": "Failed to connect to Konnected.io device"
```

Comment:
Please add already_configured as abort reason string here. Use a common reference as value.

Reply:
Wait, this is changing the options flow strings and not the config flow strings. That's not correct.

Reply:
Sorry I was quick on the commit there and didn't see this before pushing, I'll add that now as well.

Reply:
Please move `connection_error` up to the config flow abort strings. We already have a string for already_configured there.

I suggest also changing to use the common string `cannot_connect`. That's standard.

Reply:
Lol, I should slow down so I can get this right. I'll wait a few mins before pushing any more 😄.

Reply:
I appreciate your patience, sorry for being a pain.